server/service.go

```diff
@@ -102,15 +125,22 @@ func (s *Service) Stop() error {
 	for i := 1; i < len(stopables); i++ {
 		stopOrder[i] = len(stopables) - i
 	}
-
-	protocol.Debug("service: stopping %d modules in this order relative to registration: %v", len(stopOrder), stopOrder)
+	log.WithFields(log.Fields{
+		"module":        "service",
+		"no_of_modules": len(stopOrder),
```

Comment:
camel case fix please. `modules` and `order`. Message says clearly that it's stopping.